modules/common/src/main/java/com/opengamma/sdk/common/ServiceInvoker.java

```diff
@@ -43,6 +43,10 @@
  * such as via the try-with-resources statement.
  */
 public final class ServiceInvoker implements AutoCloseable {
+  /**
+   * The User Agent header to attach to all outgoing HTTP requests.
+   */
+  public static final String USER_AGENT = "OG-JAVA-SDK";
```

Comment:
I'm tempted to hard code the version, until we find a proper way of extracting it using Maven (for context, when the application is packaged into a JAR, the version is stored in a metadata file. However when it's in its project structure, we'd have to run `mvn org.apache.maven.plugins:maven-help-plugin:2.1.1:evaluate -Dexpression=project.version | grep "^[^[]"` which takes many seconds. I'd say this source looks too much for what we need: https://gist.github.com/andreiruse/49c92e7239034478a8217d3058863963

@jonathansenior Regarding, other information, Java is exposing quite a bit of information in `System.getProperties()`. 
So far, we're looking at something like: `opengamma-sdk-java/1.1.0-SNAPSHOT (Mac OS X 10.12.4 x86_64) (Oracle Corporation 1.8.0_112)`. Does this look ok?